README.md

```diff
@@ -83,8 +83,7 @@ Please see the [User Guide](https://docs.howso.com/user_guide/index.html) for ba
 - Trainee editing
 - ID-based privacy
 
-There is also a set of basic [Python examples](https://docs.howso.com/examples/index.html) that provide a complete set of Python scripts and [Jupyter notebooks](https://jupyter.org/) to run.
-
+There is also a set of basic [Jupyter notebooks](https://jupyter.org/) to run that provides a complete example of how to use Howso Engine.
```

Comment:
I was thinking we link https://docs.howso.com/examples/index.html to one of these terms. Either replacing the jupyter link or link to the the word example/something like that

Reply:
Link added and slight re-wording.